src/main/java/seedu/multitasky/logic/parser/ArgumentMultimap.java

```diff
@@ -60,4 +60,20 @@ public void put(Prefix prefix, String argValue) {
         return getValue(new Prefix(""));
     }
 
+    // @@author A0125586X
+    /**
+     * Method to extract an ArrayList of specified prefixes that are present in the ArgumentMultimap
+     */
+    public ArrayList<String> getPresentPrefixes(String... prefixes) {
+        ArrayList<String> presentPrefixes = new ArrayList<>();
+        for (String prefix : prefixes) {
+            if (getAllValues(new Prefix(prefix)).size() != 0) {
+                presentPrefixes.add(prefix);
+            } else {
+                // continue
+            }
```

Comment:
I think there's no need for the `else` block, I originally had it as I was doing a println there while debugging, but now it's not needed anymore.